### ISSUE TITLE
GGMAX - PlaySound(e,0) problem

### ISSUE THREAD

**Issue by Necrym59** [TheGameCreators/GameGuruRepo#5191]
PlaySound(e,0) now no longer sems to play a designated sound

**Comment by LeeBamberTGC** [comment]
@Necrym59 Does the out-of-the-box sound zone game element work with the default trigger audio?

**Comment by Necrym59** [comment]
@LeeBamberTGC Sorry Lee, was trying to playsound then 'e' entity died (Health=0) so sound stops. Is there a way to play sound but as/when entity dies the sound plays through to end of sound file.

**Comment by LeeBamberTGC** [comment]
@Necrym59 Yes you need to load, play and handle a 'global sound' (i.e. global.lua: PlayGlobalSound) as this survives beyond the existence of an object script running :)

**Comment by Necrym59** [comment]
@LeeBamberTGC Thanks Lee, but can the filename selected into <sound0> selection box be used to transfer the filename into LoadGlobalSound(filename, iID)

**Comment by LeeBamberTGC** [comment]
@Necrym59 You can use this command:

GetEntityString : GetEntityString ( e, slot ) -- where e is the entity number and slot (0-6) is the sound slot index (soundslot 4 is not used)

**Comment by Necrym59** [comment]
Thanks Lee - that worked perfectly for what i needed.
